### ISSUE TITLE
.NET Core support

### ISSUE THREAD

**Issue by vycius** [bcemmett/SurveyMonkeyApi-v3#41]
Add supoort for .NET Core. 
```    
Package SurveyMonkeyApi 3.1.116 is not compatible with netcoreapp1.0 (.NETCoreApp,Version=v1.0). Package SurveyMonkeyApi 3.1.116 supports:
      - net40 (.NETFramework,Version=v4.0)
      - net45 (.NETFramework,Version=v4.5)
      - net451 (.NETFramework,Version=v4.5.1)
      - net452 (.NETFramework,Version=v4.5.2)
    One or more packages are incompatible with .NETCoreApp,Version=v1.0.
```

**Comment by bcemmett** [comment]
I did think briefly of doing this. The catch is that .NET Core doesn't include WebClient, so that would need to be replaced by HttpClient. Not impossible, but it's not as trivial as just telling this to build for another target.

**Comment by bcemmett** [comment]
.NET Core 2.0 reintroduced WebClient which made this possible, so #63 adds support for .NET Core 2.0, released to NuGet as v3.2.175.
